packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -293,6 +301,25 @@ export default class RunPanelStore {
             }
         );
 
+        this.reloadWorkspaceOnSwitchAccount = reaction(
```

Comment:
There's a `registerOnAccountSwitch` reaction in app-store. I think you can utilise that instead.